python/mxnet/image.py

```diff
@@ -145,30 +145,30 @@ def random_size_crop(src, size, min_area, ratio, interp=2):
 
 
 def ResizeAug(size, interp=2):
-    """Make resize shorter edge to size augumenter."""
+    """Make resize shorter edge to size augmenter."""
```

Comment:
Spoke with @Roshrini offline. This PR only has some cosmetic improvements and spelling corrections. There will another sweep of this class later. Right now this is not a high priority.